dasharo-compatibility/esp-scanning.robot

```diff
@@ -0,0 +1,100 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+# Library    ../lib/QemuMonitor.py    /tmp/qmp-socket
+# Required setup keywords:
+#    Prepare Test Suite - generic setup keyword for all tests
+# Required teardown keywords:
+#    Log Out And Close Connection - generic setup keyword for all tests,
+#    closes all connections to DUT and PiKVM
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     Prepare Required Files For Qemu
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+ESP001.001 ESP Scan with OS-specific .efi files added
+    [Documentation]    This test aims to verify that any properly added .efi
+    ...    files will have boot menu entries created for them.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    ESP001.001 not supported
+
+    Power On
+    Prepare EFI Partition With System Files
+    Power On
```

Comment:
Do you need to use `Power On` here? Why not use `Execute Reboot Command` to shut down and reboot gracefully? This kwd also already restores pikvm dut connection (if it has been present), so there is no need for the manual restoration in your kwds.

Reply:
Applied `Execute Reboot Command` in [commit a90d5e0](https://github.com/Dasharo/open-source-firmware-validation/commit/a90d5e01100174c8b883b01ae914d4693f595747#diff-c8f7a5cdc7c64ba736049ef48cf813ae918ab342659418e21d720981c53f903aR38)